lib/methods/import.js

```diff
@@ -303,23 +307,34 @@ module.exports = self => {
         // If the parked/singleton doc is not found, it should be processed as a
         // regular doc.
         if (isSingleton || parkedId) {
+          // TODO: what about singleton with localized = false, we don't have aposMode
+          // TODO: do I need that for parked page?
```

Comment:
I don't think this is a problem. aposMode in my knowledge always exists (no matter localized or not). We check for draft here because we want to trigger this (expensive) code only once per unique page (aposDocId).